### PR TITLE
Skill service testing 💻🔥

### PR DESCRIPTION
**Pull request by Evgeniy-Sinicin**

### CODE REVIEW

File: InternshipBE/Tests/Services/SkillServiceTest.cs [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+namespace Tests.Services
+{
+    public class SkillServiceTest
+    {
+        SkillFixture _fixture;
+        List<SkillDTO> _skillDtos;
+
+        IMapper _mapper;
+        Mock<IUnitOfWork> _uowMock;
+        ISkillService _sut;
```
Comment (romanstat) [inline InternshipBE/Tests/Services/SkillServiceTest.cs]: private readonly

File: InternshipBE/Tests/Services/SkillServiceTest.cs [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+using System.Linq;
+using System.Threading.Tasks;
+using Tests.Fixtures;
+using Xunit;
+
+namespace Tests.Services
+{
+    public class SkillServiceTest
+    {
+        SkillFixture _fixture;
```
Comment (romanstat) [inline InternshipBE/Tests/Services/SkillServiceTest.cs]: _skillFixture

File: InternshipBE/Tests/Services/SkillServiceTest.cs (hunk 1 above)
Comment (romanstat) [inline InternshipBE/Tests/Services/SkillServiceTest.cs]: _skillService

**Review by romanstat — APPROVED** [review]

File: InternshipBE/Tests/Services/SkillServiceTest.cs (hunk 1 above)
Comment (Evgeniy-Sinicin) [reply InternshipBE/Tests/Services/SkillServiceTest.cs]: Oops, I forgot it 🙃

**Review by ihhyy — APPROVED** [review]

**Review by Xodoc — APPROVED** [review]
🙊🍌🙈🗿

**Pull request merged**